SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchList.tsx

```diff
@@ -173,13 +173,13 @@ export default function EventSearchList(props: IProps) {
                 defaultColumns={["Event Type"]}
             />
             {status == 'loading' ? null :
-                    data.length >= 100 ?
-                        <div style={{padding: 10, backgroundColor: '#458EFF', color: 'white'}}>
-                            Only the first {data.length} results are shown - please narrow your search
-                        </div> :
-                        <div style={{padding: 10, backgroundColor: '#458EFF', color: 'white'}}>
-                            {data.length} results
-                        </div>}
+                data.length >= parseInt(localStorage.getItem('SEBrowser.Settings.numberResults')) ?
```

Comment:
Should we replace this with loading in from the slice instead of local storage and only have the local storage be accessed by the slice?